std_msgs/Header header
int32 anchor_id
geometry_msgs/Point[] points  # Point cloud at anchor location for map building